changes/ticket33072

```diff
@@ -0,0 +1,4 @@
+  o Minor bugfix (directory authority):
+    - Always allow compressed directory requests and introduce option
```

Comment:
Oh, the changes file is out of sync as well.

We let AuthDirRejectRequestsUnderLoad and the bandwidth buckets decide about compressed requests.